pkg/proxy/fixtures/gencerts.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/bash
 
+set -euo pipefail
+
 if ! [[ "$0" =~ "./gencerts.sh" ]]; then
   echo "must be run from 'fixtures'"
   exit 255
 fi
 
-if ! which cfssl; then
+if ! command -v cfssl; then
   echo "cfssl is not installed"
+  echo "use: go install -mod mod github.com/cloudflare/cfssl/cmd/cfssl"
```

Comment:
It still has "`-mod mod`"
```suggestion
  echo "use: go install github.com/cloudflare/cfssl/cmd/cfssl"
```